.github/workflows/ci_test-base.yml

```
@@ -30,30 +33,17 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Update Pip
-      run: |
-        pip install "pip>=20.1" --upgrade --user  # needed for get pip cacher folder
-
     # Github Actions: Run step on specific OS: https://stackoverflow.com/a/57948488/4521646
     - name: Setup macOS
       if: runner.os == 'macOS'
       run: |
         brew install libomp  # https://github.com/pytorch/pytorch/issues/20030
 
-    # Note: This uses an internal pip API and may not always work
-    # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-    - name: Get pip cache
-      id: pip-cache
-      run: |
-        python -c "from pip._internal.locations import USER_CACHE_DIR; print('::set-output name=dir::' + USER_CACHE_DIR)"
-
-    - name: Cache pip
-      uses: actions/cache@v2
+    - uses: ./.github/actions/cashing
```

Should be `caching`(folder name will need to change too)
```suggestion
    - uses: ./.github/actions/caching
```